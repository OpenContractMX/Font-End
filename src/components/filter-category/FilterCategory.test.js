import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import { FilterCategory } from "./";

it("render content", () => {
  const filter = { category: "Categoria", year: "Año" };
  const component = render(<FilterCategory filter={filter} />);

  component.getByText("Filtrar:");
  component.getByText("Categoria:");
  component.getByText("Año:");
  component.debug();
});
