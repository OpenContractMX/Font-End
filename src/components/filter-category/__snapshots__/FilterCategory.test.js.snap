// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render content 1`] = `
<DocumentFragment>
  <section
    class="filter-category"
  >
    <h2
      class="filter-category--title"
    >
      Filtrar:
    </h2>
    <div
      class="filter-category__forms"
    >
      <form
        class="filter-category__forms--filter-category"
        for="filter-category"
      >
        <label>
          Categoria:
        </label>
        <select
          class="filter-category__forms--filter-category--select-category"
        >
          <option
            value="Categoria"
          >
            Categoria
          </option>
          <option
            value="seguridad"
          >
            seguridad
          </option>
          <option
            value="salud"
          >
            salud
          </option>
          <option
            value="energia"
          >
            energia
          </option>
          <option
            value="economia"
          >
            economia
          </option>
          <option
            value="gobernacion"
          >
            gobernacion
          </option>
          <option
            value="medio_ambiente"
          >
            medio_ambiente
          </option>
          <option
            value="educacion"
          >
            educacion
          </option>
          <option
            value="comunicacion_transporte"
          >
            comunicacion_transporte
          </option>
          <option
            value="social"
          >
            social
          </option>
          <option
            value="investigacion"
          >
            investigacion
          </option>
          <option
            value="trabajo"
          >
            trabajo
          </option>
        </select>
      </form>
      <p>
        Y
      </p>
      <form
        class="filter-category__forms--filter-year"
        for="filter-year"
      >
        <label>
          Año:
        </label>
        <select
          class="filter-category__forms--filter-year--select-year"
        >
          <option
            value="Año"
          >
            Año
          </option>
          <option
            value="2015"
          >
            2015
          </option>
          <option
            value="2016"
          >
            2016
          </option>
          <option
            value="2017"
          >
            2017
          </option>
          <option
            value="2018"
          >
            2018
          </option>
        </select>
      </form>
    </div>
  </section>
</DocumentFragment>
`;
